copy.go

```diff
@@ -160,28 +165,26 @@ func Copy(ctx context.Context, src ReadOnlyTarget, srcRef string, dst Target, ds
 // CopyGraph copies a rooted directed acyclic graph (DAG) from the source CAS to
 // the destination CAS.
 func CopyGraph(ctx context.Context, src content.ReadOnlyStorage, dst content.Storage, root ocispec.Descriptor, opts CopyGraphOptions) error {
+	// if Concurrency is not set or invalid, use the default concurrency
+	if opts.Concurrency <= 0 {
+		opts.Concurrency = defaultConcurrency
+	}
 	// use caching proxy on non-leaf nodes
 	if opts.MaxMetadataBytes <= 0 {
 		opts.MaxMetadataBytes = defaultCopyMaxMetadataBytes
 	}
 	proxy := cas.NewProxyWithLimit(src, cas.NewMemory(), opts.MaxMetadataBytes)
-	return copyGraph(ctx, src, dst, proxy, root, opts)
+	return copyGraph(ctx, src, dst, root, proxy, semaphore.NewWeighted(int64(opts.Concurrency)), status.NewTracker(), opts)
```

Comment:
Based on https://github.com/oras-project/oras-go/pull/397/files#r1065821344, we can do
```suggestion
	return copyGraph(ctx, src, dst, root, nil, nil, nil, opts)
```